specification/schemas/Organization.json

```diff
@@ -23,6 +23,37 @@
         "iban": {
           "$ref": "#/components/schemas/Iban"
         },
+        "language": {
+          "type": "enum",
```

Comment:
In json schema, `enum` is not a type, but any type can be an enum. In this case the type should be `string`.